README.md

```diff
@@ -44,6 +44,7 @@ they have been replaced in this way:
 - material/bottom_navigation_bar.shifting_transition.2.7.png
 - packages/flutter/test/material/text_field_cursor_test.0.0.png
 - packages/flutter/test/material/text_field_cursor_test.1.0.png
+- packages/flutter/test/material/text_field_opacity_test.0.2.png
```

Comment:
According to this, this file is now obsolete and can be deleted, but that is the name of the reference image you are checking in. Do you need to increment the version number and check in a new file instead?

Reply:
Whoops, thanks for catching that!

Yes I think I need to check in `text_field_opacity_test.0.3.png`.  Will do that.